app/form.js

```diff
@@ -0,0 +1,40 @@
+/* file: form.js
+ * authors: H. Bouakaz, S. de Vliet, S. de Jong & E. Werkema
+ * date: 19/3/2016
+ * version 1.1
+ *
+ * Description:
+
+ */
+
+define(['app/database', 'jquery', 'bootstrap', 'parsley', 'bootstrap-select'], function (db, $, bootstrap, parsley, select) {
+	 var numberOfFormItems = 1;
+   $(".selectpicker").selectpicker();
+	 $(".add").click(function(add) {
+     numberOfFormItems++;
+	   var newElement = $('#item-layout').clone(true).appendTo("#items table").removeAttr("id");
+     newElement.children("td:nth-child(1)").html(numberOfFormItems);
+     newElement.children("td:nth-child(2) input").attr("name", "question"+numberOfFormItems).addClass("question"+numberOfFormItems);
+     newElement.children("td:nth-child(3) input").attr("name", "answer"+numberOfFormItems).addClass("answer"+numberOfFormItems);
+     newElement.children("td:nth-child(4) input").attr("name", "hint"+numberOfFormItems).addClass("hint"+numberOfFormItems);
+
+     return false;
+	 });
+
+	 $(".remove").click(function() {
+	  if (numberOfFormItems != 1) {
```

Comment:
Wouldn't it be better to use numberOfFormItems < 1, just in case it somehow becomes smaller than 1 by accident?